core/src/main/resources-filtered/org/pentaho/platform/dataaccess/datasource/wizard/public/modeler.properties

```diff
@@ -142,6 +142,14 @@ validation.level.MISSING_LAT=Level must have a Latitude member property
 validation.level.MISSING_LON=Level must have a Longitude member property
 default_aggregation=Default Aggregation
 
+# Model Tree alternative text
+modeler.alternative_text.model=Model
```

Comment:
did not add the related properties for the internationalization files for English, Japanese, etc.. (ex:  modeler_en.properties, modeler_jp.properties). 

I was told PM will make this decision at the end of release and an magic process will happen